CMakeLists.txt

```diff
@@ -60,7 +59,7 @@ add_library(msdfgen::msdfgen ALIAS msdfgen)
 set_target_properties(msdfgen PROPERTIES PUBLIC_HEADER "${msdfgen_HEADERS}")
 target_include_directories(msdfgen INTERFACE
 	$<INSTALL_INTERFACE:include>
```

Comment:
this is also wrong if I understand that others projects should do `#include <msdfgen.h>`

Reply:
@Chlumsky what do you think about this ? Do you prefer allowing both ways of including the lib or only allowing `<msdfgen.h>` and making a breaking update ?

Reply:
Why is it also wrong other than breaking backwards compatibility for the include?